code/game/machinery/syndicatebomb.dm

```diff
@@ -274,6 +274,17 @@
 /obj/item/weapon/bombcore/badmin/explosion/detonate()
 	explosion(get_turf(src),HeavyExplosion,MediumExplosion,LightExplosion, flame_range = Flames)
 
+/obj/item/weapon/bombcore/miniature
+	name = "small bomb core"
+	w_class = 2
+
+/obj/item/weapon/bombcore/miniature/detonate()
+	if(adminlog)
+		message_admins(adminlog)
+		log_game(adminlog)
+	explosion(src.loc,1,2,4,flame_range = 2) //Identical to a minibomb

```

Comment:
The explosion proc only logs that an explosion occured. It's usually useful to know who and/or what caused it.